assets/src/components/higher-order/with-amp-story-settings.js

```diff
@@ -261,6 +297,43 @@ export default createHigherOrderComponent(
 							</RotatableBox>
 						</ResizableBox>
 					) }
+					{ ! ( isLast && isFirst ) && (
+						<InspectorControls>
+							<PanelBody
+								className="amp-story-order-controls"
+								title={ __( 'Block position', 'amp' ) }
+							>
+								{ ! isLast && (
+									<IconButton
+										className="amp-story-controls-bring-forward"
+										onClick={ bringForward }
+										icon={ bringForwardIcon( { width: 24, height: 24 } ) }
+									>{ __( 'Forward', 'amp' ) }</IconButton>
+								) }
+								{ ! isLast && (
+									<IconButton
+										className="amp-story-controls-bring-front"
+										onClick={ moveFront }
+										icon={ bringFrontIcon( { width: 24, height: 24 } ) }
+									>{ __( 'Front', 'amp' ) }</IconButton>
+								) }
+								{ ! isFirst && (
+									<IconButton
+										className="amp-story-controls-send-backwards"
+										onClick={ sendBackward }
+										icon={ sendBackwardIcon( { width: 24, height: 24 } ) }
+									>{ __( 'Backward', 'amp' ) }</IconButton>
+								) }
+								{ ! isFirst && (
+									<IconButton
+										className="amp-story-controls-send-back"
+										onClick={ moveBack }
+										icon={ sendBackIcon( { width: 24, height: 24 } ) }
+									>{ __( 'Back', 'amp' ) }</IconButton>
```

Comment:
The `IconButton` components in the block mover had `label`, `aria-describedby` and `aria-disabled` attributes. These ones here don't.

The buttons should be properly accessible, have tooltips, and be disabled when the block cannot be moved further forward/backward (i.e. is the last or first in list)

Reply:
Yes, I think that disabling makes more sense, thanks!

Reply:
Added within e5e58aa. Note: AFAIK the tooltip only shows on hover if there is no content in the `IconButton`.